SecurityScorecard/vulnerability_assessment_scanner/ingest/parser.yml

```diff
@@ -0,0 +1,39 @@
+name: vulnerability_assessment_scanner
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: set_common_fields
+
+stages:
+  set_common_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parsed_event.message.created_at}}"
```

Comment:
I think we can put the timestamp in the pipeline in case there was some changes in the time.

Reply:
yes it make sense! thank you!